setCredentials is required but not mentioned in documentation when using setServiceAccountId

* Library version: 6.6.0
  * Firebase Product: firebase-admin

Following the [documentation](https://firebase.google.com/docs/auth/admin/create-custom-tokens) under "Using a service account ID" I should be able to skip the `setCredentials` option in the builder, as based on the documentation it's assumed to replace the need for a local copy of `service-account.json`. However should the `setCredentials` be missed, a NullPointerException is thrown 
```java
Caused by: java.lang.NullPointerException: FirebaseOptions must be initialized with setCredentials().
```
Attempt to set FirebaseOptions without a `setCredentials` option. 
```java
public FirebaseAuth firebaseAuth() throws IOException {
    FirebaseOptions options = new FirebaseOptions.Builder()
            .setServiceAccountId(SERVICE_ACCOUNT_EMAIL)
            .setProjectId(PROJECT_ID)
            .setDatabaseUrl(String.format("https://%s.firebaseio.com/", DATABASE_NAME))
            .build();

    FirebaseApp.initializeApp(options);

    return FirebaseAuth.getInstance();
}
```
Is the documentation incorrect/lacking detail or is this a bug?

--- a/src/main/java/com/google/firebase/FirebaseOptions.java
+++ b/src/main/java/com/google/firebase/FirebaseOptions.java
@@ -296,7 +296,8 @@ public final class FirebaseOptions {
     }
 
     /**
-     * Sets the <code>GoogleCredentials</code> to use to authenticate the SDK.
+     * Sets the <code>GoogleCredentials</code> to use to authenticate the SDK. This parameter
+     * must be specified when creating a new instance of {@link FirebaseOptions}.
      *
      * <p>See <a href="https://firebase.google.com/docs/admin/setup#initialize_the_sdk">
      * Initialize the SDK</a> for code samples and detailed documentation.

--- a/src/test/java/com/google/firebase/snippets/FirebaseAppSnippets.java
+++ b/src/test/java/com/google/firebase/snippets/FirebaseAppSnippets.java
@@ -121,9 +121,10 @@ public class FirebaseAppSnippets {
     // [END access_services_nondefault]
   }
 
-  public void initializeWithServiceAccountId() {
+  public void initializeWithServiceAccountId() throws IOException {
     // [START initialize_sdk_with_service_account_id]
     FirebaseOptions options = new FirebaseOptions.Builder()
+        .setCredentials(GoogleCredentials.getApplicationDefault())
         .setServiceAccountId("my-client-id@my-project-id.iam.gserviceaccount.com")
         .build();
     FirebaseApp.initializeApp(options);

